code/_onclick/item_attack.dm

```diff
@@ -213,6 +216,13 @@ avoid code duplication. This includes items that may sometimes act as a standard
 //Area of effect attacks (swinging)
 /obj/item/proc/tileattack(mob/living/user, turf/targetarea, modifier = 1)
 	var/original_force = force
+	var/original_unwielded_force = force
+	if(!wielded)
+		return
+	if(force_wielded_multiplier)
+		original_unwielded_force = force/force_wielded_multiplier
+	else
+		original_unwielded_force = force/1.3	
```

Comment:
```suggestion
	if(!wielded)
		return
	var/original_force = force
	var/original_unwielded_force = force_wielded_multiplier ? force / force_wielded_multiplier : force / 1.3
```
Is it supposed to return null if the weapon isn't wielded? If it is, you could probably clean this up further. If not, this proc may not work as intended.

Reply:
yeah, it's supposed to check for this, so if the weapon embeds in someone on the first hit of a wide swing attack, the rest of the swing attack doesn't deal damage.

Reply:
Right, if it leaves your hands it becomes unwielded. Probably fine as is in the suggestion, then.